packages/terra-application-navigation/tests/jest/mockIntl.js

```diff
@@ -0,0 +1,10 @@
+export default {
```

Comment:
can we use [terra-enzyme-intl](https://github.com/cerner/terra-enzyme-intl)

Reply:
I believe the mock is required as a result of directly consuming context when using react hooks.